primitives/linux-port/primitiveprocessor.h

```diff
@@ -329,6 +329,7 @@ class PrimitiveProcessor
     int dict_OffsetIndex, currentOffsetIndex;		// used by p_dictionary
     int fDebugLevel;
     dbbc::Stats* fStatsPtr; // pointer for pmstats
+    // To be removed b/c always true
     bool logicalBlockMode;
```

Comment:
Are we removing this var in this branch?

Reply:
No, not yet b/c it is also change of the default.

Reply:
I plan to remove it merging column scanning code contribution later.